addon/authenticators/oauth2-password-grant.js

```diff
@@ -31,6 +32,18 @@ const keys = Object.keys || emberKeys; // Ember.keys deprecated in 1.13
   @public
 */
 export default BaseAuthenticator.extend({
+  init() {
+    this._super(...arguments);
+    deprecate(`Ember Simple Auth: Client ID as Authorization Header is deprecated in favour of Client ID as Query String Parameter.`,
+      false,
+      {
+        id: 'ember-simple-auth.OAuth2PasswordGrantAuthenticator',
```

Comment:
The deprecation ID should be specific to the deprecation - something like `ember-simple-auth.oauth2-password-grant-authenticator.client-id-as-authorization`.

Reply:
Done.